### DISCUSSION TITLE
Can we use api code instead to login?

### DISCUSSION THREAD

**Discussion by camo200sx** [llm-workflow-engine/llm-workflow-engine#161] [Q&A]
My company block the ChatGPT website. I thought I'd get around it with this but it directs me to sign in. Any chance to use API token instead somehow. This may not even be a thing. Thought I'd ask... 😁

**Comment by thehunmonkgroup** [comment]
This would need sponsorship.

**Comment by thehunmonkgroup** [comment]
Browser backend is deprecated, no new features specific to the backend will be added.
